Mathlib/Algebra/Order/Star/Basic.lean

```diff
@@ -111,7 +111,7 @@ lemma of_nonneg_iff [NonUnitalRing R] [PartialOrder R] [StarRing R]
     (h_nonneg_iff : ∀ x : R, 0 ≤ x ↔ x ∈ AddSubmonoid.closure (Set.range fun s : R => star s * s)) :
     StarOrderedRing R where
   le_iff x y := by
-    haveI : CovariantClass R R (· + ·) (· ≤ ·) := ⟨fun _ _ _ h => h_add h _⟩
+    haveI : AddLeftMono R := ⟨fun _ _ _ h => h_add h _⟩
```

Comment:
I don't think `haveI` should be necessary either here or below (`have` should be enough), but that's perhaps out of scope.